Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Art/Materials/Colors/Blue.mat

```diff
@@ -0,0 +1,171 @@
+%YAML 1.1
```

Comment:
We already have a blue (and other colors) material. Can we use those instead? https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MRTK/StandardAssets/Materials/MRTK_Standard_Blue.mat